gem/management/commands/download_sections.py

```diff
@@ -0,0 +1,28 @@
+import json
+from django.core.management.base import BaseCommand
+from molo.core.models import SectionPage
+
+
+class Command(BaseCommand):
+
+    def handle(self, *args, **options):
+        data = {}
+        sections = SectionPage.objects.all()
+
+        for section in sections:
+            section_data = {}
+            section_data["title"] = section.title
+            section_data["slug"] = section.slug
+            section_data["locale"] = section.language.locale
+            section_data["live"] = section.live
+            section_data["uuid"] = section.uuid
+            section_data["translation_pks"] = [page.pk for page in section.translated_pages.all()]
+            section_data["description"] = section.description
+            if section.image:
+                section_data["image_name"] = section.image.title
```

Comment:
I think you need the image url here as well

Reply:
if there is an image in the richtext or image block in the body, it gets added automatically, with all it's details

Reply:
I think you were responding to my other comment? this one is about copying the url for the image used on the Section page